com.abisource.AbiWord.json

```diff
@@ -0,0 +1,218 @@
+{
+    "app-id" : "com.abisource.AbiWord",
+    "branch" : "3.0",
+    "runtime" : "org.gnome.Sdk",
+    "runtime-version" : "3.28",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "abiword",
+    "tags" : [
+       "stable"
+    ],
+    "finish-args" : [
+        "--allow=devel",
+        "--talk-name=org.freedesktop.Flatpak",
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--filesystem=host",
+        "--share=network",
+        "--talk-name=org.gtk.vfs.*"
+    ],
+    "build-options" : {
+        "env" : {
+            "V" : "1"
+        }
+    },
+    "cleanup" : [
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+            "name" : "boost",
+            "config-opts" : [
+                "--with-toolset=gcc",
+                "--with-icu"
+            ],
+            "cleanup" : [
+                "/lib/libboost_*.a"
+            ],
+            "sources" : [
+                {
+                    "type" : "archive",
+                    "url" : "https://downloads.sourceforge.net/project/boost/boost/1.60.0/boost_1_60_0.tar.bz2",
+                    "sha256" : "686affff989ac2488f79a97b9479efb9f2abae035b5ed4d8226de6857933fd3b"
+                },
+                {
+                    "type" : "file",
```

Comment:
Use the `simple` buildsystem instead; also build only specific libraries Abiword uses.

Reply:
Some example: https://github.com/flathub/org.qbittorrent.qBittorrent/blob/4f8e78558fd83ea4f684848e69e2f6e00f02808a/org.qbittorrent.qBittorrent.json#L28

Except now there is $FLATPAK_BUILDER_N_JOBS env var to specify number of jobs.

Reply:
I changed this now for the next push.